akvo/rsr/static/tests/scripts-src/my-results/utils.test.js

```diff
@@ -9,14 +9,191 @@
 import chai from 'chai'
 import React from 'react'
 import * as utils from '../../../scripts-src/my-results/utils'
+import * as c from '../../../scripts-src/my-results/const'
 import {shallow} from 'enzyme'
 
 let expect = chai.expect;
 
+describe("function distinct", () => {
+    it('expect to return an array with unique values', () => {
+        const arrayWithDuplicates = [1,2,3,4,4],
+              uniqueArray = [1,2,3,4];
```

Comment:
I don't mind declaring several consts at the same time this way, but I think keeping to 4 space indents here too makes sense since it otherwise sooner or later becomes a hassle to keep things "lined up".